include/chain_invoke.h

```diff
@@ -61,18 +61,38 @@ using namespace msl;
             }
         }
 
+        template<bool have_agrs, typename TRet, typename stack_args>
```

Comment:
Add doxy comment here, please, to show the intension of the data type.

Reply:
done. Possible see after upload